pkg/cgroups/systemd_linux.go

```diff
@@ -140,6 +140,12 @@ func resourcesToProps(res *configs.Resources) (map[string]uint64, map[string]str
 		uMap["CPUQuotaPerSecUSec"] = cpuQuotaPerSecUSec
 	}
 
+	if res.CpuShares != 0 {
```

Comment:
the conversion should be done only on cgroup v2.  On Cgroup v1 I think you can specify CPUShares to systemd

Reply:
ok, in that case should I add a `isv2()` check and then specify either "CPUWeight" or "CPUShares"